tests/basetest.rb

```diff
@@ -141,9 +141,16 @@ def config(*args)
     # Send the entire config as one string but be sure not to return until
     # we are safely back out of config mode, i.e. prompt is
     # 'switch#' not 'switch(config)#' or 'switch(config-if)#' etc.
-    @device.cmd('String' => "configure terminal\n" + args.join("\n") + "\nend",
-                # NX-OS has a space after '#', IOS XR does not
-                'Match'  => /^[^()]+[$%#>] *\z/n)
+    result = @device.cmd('String' => "configure terminal\n" + args.join("\n") +
+                         "\nend",
+                         # NX-OS has a space after '#', IOS XR does not
+                         'Match'  => /^[^()]+[$%#>] *\z/n)
+
+    if /invalid/i.match(result)
+      Cisco::Logger.warn("Config result:\n#{result}")
+    else
+      Cisco::Logger.debug("Config result:\n#{result}")
```

Comment:
I tried running test_interface with info vs. debug and ended up with ~600 lines of config output.  Seems too noisy even for a test.